README.md

```diff
@@ -114,39 +114,43 @@ py -3 -m venv ./.venv
    ```bash
    pipenv shell
    ```
-   (examples: `robotpy`, `robotpy[ctre,navx]`, `robotpy[all]`) (see: [robotpy on pypi](https://pypi.org/project/robotpy/))
-1. **Download python for roboRIO**
+1. **download required robotpy packages**
+   This also includes the python image, as well as all packages defined in pyproject.toml
```

Comment:
Does this even have a pyproject.toml

Reply:
with #9 it would
didn't feel like putting in an inaccurate pyproject.toml